src/widgets/helper/ChannelView.cpp

```diff
@@ -1716,6 +1716,22 @@ void ChannelView::mouseReleaseEvent(QMouseEvent *event)
             else if (hoverLayoutElement->getFlags().has(
                          MessageElementFlag::Username))
             {
+                if (event->modifiers() == Qt::ShiftModifier)
```

Comment:
Perhaps check for the Alt modifier condition here as well instead of having two blocks?